clio-server/config/docker-compose.yml.ctmpl

```diff
@@ -0,0 +1,18 @@
+{{with $app_dir := env "APP_DIR"}}
```

Comment:
It doesn't now, but it could. The env var is set in the deploy script with a default value of `"/app"`.